### PR TITLE
Update manage-feedback-policies-in-teams.md

### PR DESCRIPTION
**Pull request by SerinaJingyiLu**

### CODE REVIEW

File: Teams/manage-feedback-policies-in-teams.md [hunk 1]
```diff
@@ -62,6 +62,7 @@ To turn off and turn on the features, set the following parameters:
  - **Give feedback**: Set the **userInitiatedMode** parameter to **enabled** to allow users who are assigned the policy to give feedback. Setting the parameter to **disabled** turns off the feature and users who are assigned the policy don't have the option to give feedback.
 
  - **Surveys**: Set the **receiveSurveysMode** parameter to **enabled** to allow users who are assigned the policy to receive the survey. To have users receive the survey and allow them to opt out, set the parameter to **enabledUserOverride**. In Teams, users can then go to **Settings** > **Privacy** and choose whether they want to participate in surveys. Setting the parameter to **disabled** turns off the feature and users who are assigned the policy won't receive the survey.
+**Note**: Call quality feedback is now not gated by **receiveSurveysMode**.
```
Comment (JohanFreelancer9) [inline Teams/manage-feedback-policies-in-teams.md:65]:
```suggestion

> [!NOTE]
> Call quality feedback is now not gated by **receiveSurveysMode**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @SerinaJingyiLu, kindly commit the suggested changes. Many thanks.

**Comment by SerinaJingyiLu** [comment]
> Copy review - @SerinaJingyiLu, kindly commit the suggested changes. Many thanks.

Updated~

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@MikePlumleyMSFT 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**